### ISSUE TITLE
add single track 128x64 support

### ISSUE THREAD

**Issue by sevket-karayilan** [pixelmatix/SmartMatrix#176]
I bought a single piece, 128 columns and 64 rows, 1/32 LED matrix, but as far as I understand, the working mentality is not the same as 2x64x64. Please add this panel as well.

**Comment by ertugrul-labs** [comment]
Soruna bir çözüm buldunuz mu?

**Comment by sevket-karayilan** [comment]
Yes, I edited the library and wrote my own library code. That wasn't enough, so I also found panel(s) containing chips compatible with my code. These researches and the writing process took me 1.5 months in total.

Evet. kütphaneyi editleyerek kendi kütüphane kodumu yazdım. buda yetmedi, koduma uygun çipler içeren bir panel(ler) buldum. Bu araştırmaları ve yazması ile birlikte 1,5 ayımı aldı.

**Comment by ertugrul-labs** [comment]
Congratulations! Can you share it with us?

Bizimle paylaşım yaparmısınız hocam

**Comment by sevket-karayilan** [comment]
özelden erişir misiniz?
sevket_@hotmail.com
